test/functional/multiwallet.py

```diff
@@ -17,69 +17,78 @@ def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
         self.extra_args = [['-wallet=w1', '-wallet=w2', '-wallet=w3', '-wallet=w']]
+        self.variants = ["rpc", "cli"]
```

Comment:
BTW, I thought of:
```py
self.matrix = { mode: ['rpc', 'cli'], foo: ... }
```
Which would generate all combinations.

Reply:
What is the suggestion? That seems identical to current test except it renames things and adds a foo.

Reply:
Sorry if it was not clear. The suggestion is to support combinations. A more decent example:
```py
self.matrix = { a: ['a1', 'a2'], b: ['b1', 'b2', 'b3'] }
```
This would call 6 times `run_test`.

Reply:
Could be added when needed, imo.

Reply:
Sure, I didn't mean to do it here. Variants is enough in this PR.